dashboard/config/application.rb

```diff
@@ -37,6 +37,12 @@ class Application < Rails::Application
 
     config.middleware.insert_after Rails::Rack::Logger, VarnishEnvironment
     config.middleware.insert_after VarnishEnvironment, FilesApi
+
+    if CDO.throttle_data_apis
```

Comment:
Like mysql, Redis does run automatically in properly configured environments, but at startup rather in bin/dashboard-server.

Our Chef recipes install the redis service which auto-runs at server startup.  Likewise "apt-get /brew install redis" in our dev setup instructions causes it to run automatically.